src/sentry/api/endpoints/project_events.py

```diff
@@ -23,21 +20,10 @@ def list_project_available_samples_scenario(runner):
 class ProjectEventsEndpoint(ProjectEndpoint):
     doc_section = DocSection.EVENTS
 
-    @attach_scenarios([list_project_available_samples_scenario])
-    def get(self, request, project):
-        """
-        List a Project's Events
-        ```````````````````````
-
-        Return a list of events bound to a project.
-
-        Note: This endpoint is experimental and may be removed without notice.
-
-        :pparam string organization_slug: the slug of the organization the
-                                          groups belong to.
-        :pparam string project_slug: the slug of the project the groups
-                                     belong to.
-        """
+    def __search_events_legacy(self, request, project):
+        from sentry import quotas
+        from sentry.api.paginator import DateTimePaginator
+        from sentry.models import Event
```

Comment:
These don't need to be late imports but I figured it would be a little clearer when it comes time to remove the legacy implementation.